JustSaying/JustSayingBus.cs

```diff
@@ -216,7 +216,8 @@ private async Task PublishAsync(IMessagePublisher publisher, Message message, in
                 }
 
                 _log.LogWarning(0, ex, $"Failed to publish message {message.GetType()}. Retrying after attempt {attemptCount} of {Config.PublishFailureReAttempts}");
-                await Task.Delay(Config.PublishFailureBackoffMilliseconds * attemptCount, cancellationToken).ConfigureAwait(false);
+                var millisDelay = (int)(Config.PublishFailureBackoff.TotalMilliseconds * attemptCount);

```

Comment:
This is just an internal delay, so I don't think we really need to worry about doing the truncation here?

For 2.0 (not sure if it works in non-.NET Standard TFMs), could just do this ([docs](https://docs.microsoft.com/en-us/dotnet/api/system.timespan.op_multiply?view=netcore-2.1&viewFallbackFrom=netframework-4.7.2)):

```csharp
await Task.Delay(Config.PublishFailureBackoff * attemptCount, cancellationToken).ConfigureAwait(false);
```

Reply:
That doesn't work for me,  `Config.PublishFailureBackoff * attemptCount`  gives 
`Error CS0019 Operator '*' cannot be applied to operands of type 'TimeSpan' and 'int`, There's no "multiply" method found iether in this version of .NET.

It is truncated to int because `TotalMilliseconds` is typed as `double`, and the next line is using `Task.Delay(int millisecondsDelay`

Reply:
Fair enough regarding the `*` operator - it must not be available for 100% of our TFMs.

Why not use the overload that takes a `TimeSpan` then?

Reply:
Ah yes, that version is just as succinct, because `TimeSpan.FromMilliseconds`  _does_ accept a double value.